### PR TITLE
Better integrate the idea of "filtra-ultra low latency".

### LINKED ISSUE #29: Finer grained latency considerations
The opcons drafts uses 4 bands:

   *  ultra low-latency (less than 1 second)
   *  low-latency live (less than 10 seconds)
   *  non-low-latency live (10 seconds to a few minutes)
   *  on-demand (hours or more)
   
I think these bands lose some granularity that is useful.  For example, a functional video call or interactive game probably needs much less than 1s latency, while low latency live may span lower than 1s in some cases?

### PR DESCRIPTION
**Pull request by SpencerDawkins**
fixes #29

### CODE REVIEW

**Comment by SpencerDawkins** [comment]
@afrind - Please let me know if this seems about right. 

One thing that's worth mentioning to you and to @fiestajetsam - the latency characterizations that we wrote did not make use of the new "ultra-ultra-low latency" categories. I changed the one for gaming to ull100 ("ultra-low-latency under 100 ms") based on conversations at IETF 112, but other people would be better choices to review the characterizations of the use cases they're most interested in.

File: draft-gruessing-moq-requirements.md [hunk 1]
```diff
@@ -201,7 +215,7 @@ senders and receivers.
 
 **Senders/Receivers**: One to One
 **Bi-directional**: Yes
-**Latency**: Significantly less than "Ultra-Low"
+**Latency**: Ull100
```
Comment (afrind) [inline draft-gruessing-moq-requirements.md:218]: Is VC not also ull500 or is 500ms-1s lag acceptable/normal?

Comment (SpencerDawkins) [reply draft-gruessing-moq-requirements.md:218]: @afrind - I'm pretty sure that the number is under 500 ms, but a quick Google turns up opinions about 300 ms, and about 200 ms. I'd like to understand what the right categories are, that would cover the use cases we seem to be converging on. So I'll make this change, and keep the PR open.

File: draft-gruessing-moq-requirements.md [hunk 2]
```diff
@@ -99,6 +99,20 @@ and the goal is to provide a protocol stack that looks like this:
...
+
+- ultra low-latency (less than 1 second)
+- low-latency live (less than 10 seconds)
+- non-low-latency live (10 seconds to a few minutes)
+- on-demand (hours or more)
+
+These latency bands were appropriate for streaming media, which was the target for {{I-D.draft-ietf-mops-streaming-opcons}}, but some realtime media may have requirements that are significantly less than "ultra-low latency". Within this document, we are also using
+
+- ull500 (less than 500 ms)
+- ull100 (less than 100 ms)
```
Comment (fiestajetsam) [inline draft-gruessing-moq-requirements.md:112]: My only view on this bikeshed - I would propose we refer to these as "Sub-500" and "Sub-100" just for readability.

**Pull request merged**